src/Jackett.Common/Indexers/BakaBT.cs

```diff
@@ -81,6 +82,18 @@ private async Task DoLogin()
                 { "returnto", "/index.php" }
             };
 
+            //BakaBT for some reason, has two different login pages.
+            try
+            {
+                var loginKeyMatch = Regex.Match(loginForm.ContentString, @"name=\""loginKey\"".+value=\""(.+?)\""");
```

Comment:
Could you try to use the HTML parser instead? I think the code is more readable. 
I'm thinking in the following code but I can't test it.
```
var parser = new HtmlParser();
var dom = parser.ParseDocument(loginForm.ContentString);
var loginKey = dom.QuerySelector("input[name=\"loginKey\"]");
if (loginKey != null)
    pairs["loginKey"] = loginKey.GetAttribute("value");
```